src/services/TechRecordsService.ts

```diff
@@ -12,6 +12,9 @@ import {
 import {ISearchCriteria} from "../../@Types/ISearchCriteria";
 import {populateFields} from "../utils/ValidationUtils";
 import HTTPResponse from "../models/HTTPResponse";
+import IMsUserDetails from "../../@Types/IUserDetails";
+import { STATUS_CODES } from "http";
+import { stat } from "fs";
```

Comment:
STATUS_CODES and stat are not used, looks like might have been imported by mistake